.ci/magic-modules/welcome-contributor.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+echo > comment/pr_comment << EOF
+Hello!  I am a robot who works on Magic Modules PRs.
+
+I have detected that you are a community contributor, so your PR will be assigned to someone with a commit-bit on this repo for initial review.  They will authorize it to run through our CI pipeline, which will generate downstream PRs.
```

Comment:
Maybe add a "Thank you for your contribution" somewhere in here?

Reply:
Look good?

Reply:
Can we say "soon" instead of "within 2 business days". I'd rather beat their expectations (because they weren't expecting us to respond soon), rather than not meet the expectations we place on ourselves.

Reply:
Done, sure!